src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -925,6 +925,20 @@ public World get(final PortalCreateEvent e) {
 				return e.getWorld();
 			}
 		}, 0);
+		EventValues.registerEventValue(PortalCreateEvent.class, String.class, new Getter<String, PortalCreateEvent>() {
+			@Override
+			@Nullable
+			public String get(final PortalCreateEvent e) {
+				for (BlockState b : e.getBlocks()) { // Gets blocks prior to actual portal creation
```

Comment:
this won't support 1.12.2 and below, as 1.12.2 and below, this event returns blocks, not block state. 
Also, there is no Material.END_PORTAL_FRAME in 1.12.2

Reply:
I just scrapped it since it should be left up to the user